### ISSUE TITLE
[HW] <NAME>

### ISSUE THREAD

**Issue by gagansuri** [datastaxdevs/workshop-spring-stargate#106]
**Name:** Gagan Suri
**Email:** Gagan Suri
**Linkedin Profile:** https://www.linkedin.com/in/gagansuri

Attach the homework step I screenshots below:
-----------------------------------------

![image](https://user-images.githubusercontent.com/631890/131893117-13dcfd78-a5d1-4754-88a1-8bd008ef982f.png)
  
Attach the homework step II screenshots below:
-----------------------------------------

![image](https://user-images.githubusercontent.com/631890/131893117-13dcfd78-a5d1-4754-88a1-8bd008ef982f.png)

**Comment by gagansuri** [comment]
what is the second screenshot?

**Comment by 000** [comment]
> what is the second screenshot?

From this exercise: https://www.datastax.com/dev/spring

**Comment by SonicDMG** [comment]
Yes, the second screenshot is from the completed screen of the exercise link you posted.

**Comment by SonicDMG** [comment]
Also, once you complete the scenario (shouldn't take too long) please update your email so we can get you a badge.
